tests/config/format-test.js

```diff
@@ -529,3 +533,14 @@ async function format(originalText, originalOptions) {
 }
 
 export default runSpec;
+
+export function removeAngularBlockInfo(ast, deleted = []) {
```

Comment:
This is not correct, we supposed to have `sourceSpan` in AST nodes. 

https://github.com/prettier/prettier/blob/ebbd3ece8b41405e29268052e13b06a3c4e03b4c/src/language-html/parser-html.js#L232

And `sourceSpan` is ignored when comparing AST. https://github.com/prettier/prettier/blob/ebbd3ece8b41405e29268052e13b06a3c4e03b4c/src/language-html/clean.js#L4C4-L4C14

Reply:
Missing vistorKeys  https://github.com/prettier/prettier/blob/ebbd3ece8b41405e29268052e13b06a3c4e03b4c/src/language-html/visitor-keys.js#L1

Reply:
Thanks for your review
I have deleted removeAngularBlockInfo and added visitor keys.
currently, all tests in the local environment are correct.